app/javascript/src/components/Dashboard/Notes/List.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from 'react'
+
+import Note from './Note';
+import EditNotePane from './Pane/Edit';
+import DeleteAlert from "./DeleteAlert";
+
+const List = ({notes, fetchNotes}) => {
+
```

Comment:
Please don't add unnecessary blank lines in code unless it provides a visual segregation that helps in making the code more readable. Here this blank line is very unnecessary.